src/internal/MonoColor.h

```diff
@@ -0,0 +1,209 @@
+/*-------------------------------------------------------------------------
+MonoColor provides a color object that can be directly consumed by NeoPixelBus
+
+Written by Grzegorz Bialokziewicz (based on Michael C. Miller code).
+
+I invest time and resources providing this open source code,
+please support me by dontating (see https://github.com/Makuna/NeoPixelBus)
+
+-------------------------------------------------------------------------
+This file is part of the Makuna/NeoPixelBus library.
+
+NeoPixelBus is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as
+published by the Free Software Foundation, either version 3 of
+the License, or (at your option) any later version.
+
+NeoPixelBus is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public
+License along with NeoPixel.  If not, see
+<http://www.gnu.org/licenses/>.
+-------------------------------------------------------------------------*/
+#pragma once
+
+#include <Arduino.h>
+#include "NeoSettings.h"
+
+struct RgbColor;
+struct RgbwColor;
+struct HslColor;
+struct HsbColor;
+struct HtmlColor;
+
+// ------------------------------------------------------------------------
+// MonoColor represents a color object that is represented by only White
+// component. It contains helpful color routines to manipulate the color.
+// ------------------------------------------------------------------------
+struct MonoColor
+{
+    typedef NeoMonoCurrentSettings SettingsObject;
+
+    // ------------------------------------------------------------------------
+    // Construct a MonoColor using R, G, B, W values (0-255)
+    // ------------------------------------------------------------------------
+    MonoColor(uint8_t r, uint8_t g, uint8_t b, uint8_t w = 0);
```

Comment:
This constructor for other types like Rgb and Rgbw are meant to mirror the properties of the class.  This should just take and set W.